### PR TITLE
Shellcheck

### PR DESCRIPTION
**Pull request by InTheCloudDan**
Initially converting run.bats to shellcheck format.

Todo:
- [x] Add to travis

### CODE REVIEW

**Comment by lumjjb** [comment]
This is a good addition to the travis checks! Thanks for the contribution!

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ There is initial aarch64 support. For more information please check the [README.
 3. Install genisoimage on host `sudo apt install genisoimage`
 4. Install jq on host `sudo apt install jq`
 5. Ensure that docker is installed (docker-ce recent versions, i.e. v15 onwards)
+6. When adding shell tests `tests/integrations/run.bats` execute `make shellcheck` to lint the file
```
Comment (lumjjb) [inline README.md]: I think this should not be in the Installation steps. We should have a `dev.md` readme probably to include these steps. I've created an issue for it. In the meantime, let's remove this line. https://github.com/nabla-containers/runnc/issues/80

File: Makefile [hunk 2]
```diff
@@ -121,3 +121,8 @@ clean:
 		tests/integration/test_curl.nabla \
 		tests/integration/node_tests.iso
 	sudo make -C solo5 clean
+
+SHELLCHECK=docker run --rm -v "$(CURDIR)":/v -w /v koalaman/shellcheck
+.PHONY: shellcheck
+shellcheck:
+	$(SHELLCHECK) tests/integration/*.bats
```
Comment (lumjjb) [inline Makefile]: >tests/integration/helpers.bash

I think this file should be checked too!

File: tests/integration/helpers.bash [hunk 3]
```diff
@@ -55,10 +55,11 @@ function setup_test() {
             exit 1
             ;;
     esac
-    cd "$TEST_BUNDLE"
+    cd "$TEST_BUNDLE" || exit
 }
 
 function config_mod () {
+    # shellcheck disable=SC2002
```
Comment (lumjjb) [inline tests/integration/helpers.bash]: Hmm, what is the particular issue here?

Comment (InTheCloudDan) [reply tests/integration/helpers.bash]: It says unnecessary cat and to use a redirect, I did a quick search and found another repo doing the exact same thing we do and just ignoring it. We could probably open an issue on shellcheck about it but haven't looked that route yet.

Comment (lumjjb) [reply tests/integration/helpers.bash]: Hmm that's a valid point though, can we do `jq "$@" > config.json.new < config.json` ?

Comment (InTheCloudDan) [reply tests/integration/helpers.bash]: this is fixed with latest commit. I need to read up on shell redirects :P

**Pull request merged**